### ISSUE TITLE
Mixing content of columns in a forum

### ISSUE THREAD

**Issue by climberwoodi** [weblyzard/inscriptis#29]
For some websites inscriptis is mixing content of columns. 
Example: https://bpdfamily.com/message_board/index.php?topic=343886.0

The first post of the text displayed by inscriptis:
_I don't even know where/how to start so I apologize if this is a bit rambly before it makes a point but I feel the need to preface certain things. **What is your sexual orientation: Straight**                                                                                          I've never been good with women and because of that I have chronic self-esteem issues, I wouldn't say they are generally debilitating but I certainly have issues when it comes to courting women. **Who in your life has "personality" issues: Romantic partner**                                                                        I've had 2 real relationships in my life:_

**Comment by AlbertWeichselbraun** [comment]
This is actually the expected behavior since the content is aligned in a table with the bold text in the first and the italic text in the second column.

Do you think that providing an option for replacing spaces with a table separator character such as `\t` would be helpful in the context of you application?

**Comment by climberwoodi** [comment]
Yes, replacing the table separator with \t would be very helpful in the context of my application.
